uuid/uuid.go

```diff
@@ -27,6 +27,14 @@ func IsAndroid(uuid string) bool {
 	return IsValidFast(uuid) && matchUuidRegexAndroid(uuid) && !isTest(uuid)
 }
 
+func IsMaid(uuid string) bool {
+	return !IsNoid(uuid)
+}
+
+func IsNoid(uuid string) bool {
+	return uuid == "" || uuid == "00000000-0000-0000-0000-000000000000"
```

Comment:
are these the only ones used? Just curious, we can add more cases here later if necessary

Reply:
yes, we can add more cases later but only these cases are in the pitch